lib/components/tab.js

```diff
@@ -124,12 +124,7 @@ export default class Tab extends Component {
         }
       },
 
-      hasActivity: {
-        color: '#50E3C2',
-        ':hover': {
-          color: '#50E3C2'
-        }
-      },
+      hasActivity: getHeaderColor(this.props.hasActivity),
```

Comment:
I tested your branch and this line produces an error when I open a tab:
![capture d ecran 2017-05-20 a 00 10 47](https://cloud.githubusercontent.com/assets/4137761/26268748/02f294dc-3cf1-11e7-9044-4587a71c1dc8.png)

`this.props` seems to be `undefined` in this `styles` function. It seems to be a empty/non-instance `Tab` component.
I don't know what I'm doing wrong.